dev/com.ibm.ws.opentracing.1.1/resources/com/ibm/ws/opentracing/resources/Opentracing.nlsprops

```diff
@@ -0,0 +1,84 @@
+# the parameter that is inserted is a date.
+#
+# The 0 indicates that this is where the first parameter is
+# inserted in the string.  There may be more than one parameter
+# in any one string.  The 'date' tells java to format this
+# parameter as a date.
+#
+# The {0,date} code should be placed at the appropriate position
+# in the translated text.  Please note that the code in the
+# translation must match exactly the code in the original
+# string.
+#
+# The developer should describe in comments above the string to be
+# translated each of the parameters so that you will be able to
+# determine where the best location for each parameter in the
+# translated string.
+#
+# Lines that begin with a pound sign (#) are comments and should not be
+# translated.
+# -----------------------------------------------------------------------
+# This file follows the WebSphere Message Guidelines.
+# For more information, visit: 
+# http://washome.austin.ibm.com/xwiki/bin/view/MessagesTeam/
+#
+
+#-----------------------------------------------------------------------------------------------------------------------------
+# Messages used by Opentracing Impl -  range is 0000-4999, plus 9999 for the emergency message
+#-----------------------------------------------------------------------------------------------------------------------------
+OPENTRACING_NO_APPNAME_FOUND_IN_JNDI=CWMOT0000E: Lookup of the application name in JNDI did not return a value. A default name is used for Opentracing service name.
+OPENTRACING_NO_APPNAME_FOUND_IN_JNDI.explanation=The feature provides the application name to the user supplied OpentracingTracerFactory implementation. The application name could not be retrieved by JNDI lookup of "java:app/AppName". A default service name was provided.
+OPENTRACING_NO_APPNAME_FOUND_IN_JNDI.useraction=Determine why the application name is not available through JNDI lookup of "java:app/AppName".
+
+OPENTRACING_COULD_NOT_CREATE_TRACER=CWMOT0001E: Invocation of user supplied OpentracingTracerFactory.newInstance(...) method failed with Exception. Message = {0}
+OPENTRACING_COULD_NOT_CREATE_TRACER.explanation=The feature constructs Tracer implementations with a user provided OpentracingTracerFactory. The OpentracingTracerFactory.newInstance method returned an exception.
+OPENTRACING_COULD_NOT_CREATE_TRACER.useraction=Determine why the user supplied OpentracingTracerFactory newInstance method returned an exception.
+
+OPENTRACING_TRACERFACTORY_RETURNED_NULL=CWMOT0006E: Invocation of user supplied OpentracingTracerFactory.newInstance(...) method returned null
+OPENTRACING_TRACERFACTORY_RETURNED_NULL.explanation=The feature constructs Tracer implementations with a user provided OpentracingTracerFactory. The OpentracingTracerFactory.newInstance method returned null.
+OPENTRACING_TRACERFACTORY_RETURNED_NULL.useraction=Determine why the user supplied OpentracingTracerFactory newInstance method returned null.
```

Comment:
OPENTRACING_TRACERFACTORY_RETURNED_NULL=CWMOT0006E: Invocation of **a** user supplied OpentracingTracerFactory.newInstance(...) method returned null

OPENTRACING_TRACERFACTORY_RETURNED_NULL.explanation=The feature constructs Tracer implementations with a user provided **value for** OpentracingTracerFactory. *The OpentracingTracerFactory.newInstance method returned null.* **Should we include info about what the method should return?**

OPENTRACING_TRACERFACTORY_RETURNED_NULL.useraction=Determine why the user supplied OpentracingTracerFactory newInstance method returned null. **How does the developer accomplish this? Is it by examining their own work or by reaching out to the user? Are there any resources to direct them to?**